pkg/routeagent_driver/handlers/ovn/ovn_logical_routes.go

```diff
@@ -0,0 +1,128 @@
+/*
+SPDX-License-Identifier: Apache-2.0
+
+Copyright Contributors to the Submariner project.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ovn
+
+import (
+	"strings"
+
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/libovsdbops"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/nbdb"
+	"github.com/pkg/errors"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/utils/ptr"
+)
+
+const (
+	ovnClusterRouter     = "ovn_cluster_router"
+	ovnRoutePoliciesPrio = 20000
+)
+
+func (c *ConnectionHandler) ReconcileOvnLogicalRouterStaticRoutes(remoteSubnets sets.Set[string],
```

Comment:
We don't need this exported.

```suggestion
func (c *ConnectionHandler) eeconcileOvnLogicalRouterStaticRoutes(remoteSubnets sets.Set[string],
```

Reply:
done.